spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -1190,8 +1190,7 @@ function timeoutPacket(
         // ordered channel: check that packet has not been received
         // only allow timeout on next sequence so all sequences before the timed out packet are processed (received/timed out)
         // before this packet times out
-        abortTransactionUnless(nextSequenceRecv == packet.sequence)
-
+        abortTransactionUnless(packet.sequence >= nextSequenceRecv)
```

Comment:
Again I think this should remain as is. It would enforce that the only packet that gets timed out is the first in line